client/src/modules/dashboard/Venues/components/VenueForm.tsx

```diff
@@ -112,24 +123,69 @@ const VenueForm: React.FC<VenueFormProps> = (props) => {
     country: venue?.country ?? '',
     latitude: venue?.latitude ?? undefined,
     longitude: venue?.longitude ?? undefined,
+    chapter_id: chapterId,
   };
   const {
+    formState: { isDirty },
+    getValues,
     handleSubmit,
     register,
-    formState: { isDirty },
+    setValue,
   } = useForm<VenueFormData>({
     defaultValues,
   });
 
+  interface Chapter {
+    id: number;
+    name: string;
+  }
+  let adminedChapters: Chapter[] = [];
+  if (isChaptersDropdownNeeded) {
+    const { user } = useAuth();
+    adminedChapters = user?.admined_chapters ?? [];
+    if (adminedChapters.length) {
+      setValue('chapter_id', adminedChapters[0].id);
+    }
```

Comment:
This way of using `setValue` creates a kinda weird bug.  The first selection from the dropdown is ignored.  Subsequent clicks work as expected.

As I was digging into that I wondered, do we need to use `getValue` or `setValue` at all?  If we set the default value to be `chapterId` if it exists or `adminedChapters[0].id` otherwise, it should Just Work.

Reply:
I'm having problem with the default value not getting picked up when navigating to the `/dashboard/venues/new` via UI (Dashboard -> Venues -> Add new). When accessing page directly, or after refresh, the default value is selected.